sdk/communication/communication-network-traversal/swagger/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Communication Services Configuration Module
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: azure-communication-network-traversal
+override-client-name: NetworkRelayRestClient
+description: Communication Network Traversal Client
+package-version: 1.0.0-beta.1
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../src/generated
+tag: package-2021-02-22-preview1
+require:
+- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/60be518b4fa1a9fb011a0cb69ae7ca3e1cee06b1/specification/communication/data-plane/Microsoft.CommunicationServicesTurn/readme.md
+model-date-time-as-string: false
+optional-response-headers: true
+payload-flattening-threshold: 10
+use-extension:
+  "@autorest/typescript": "6.0.0-dev.20200623.2"
+add-credentials: false
+azure-arm: false
+directive:
+  from: swagger-document
+  where: '$.paths["/turn/{id}/:issueCredentials"].post'
+  transform: >
+    $["operationId"] = "CommunicationNetworkTraversal_IssueTurnCredentials";
```

Comment:
We can use the generated model types and rename the response with a transform. Try adding this and re-generate.

```
### Rename `CommunicationTurnCredentialsResponse` to `CommunicationRelayConfiguration`
custom-types-subpackage: models
custom-types: CommunicationRelayConfiguration
directive:
  - rename-model:
      from: CommunicationTurnCredentialsResponse
      to: CommunicationRelayConfiguration
```

Reply:
Tried it like above and also like this

, ```yaml
package-name: azure-communication-network-traversal
override-client-name: NetworkRelayRestClient
description: Communication Network Traversal Client
package-version: 1.0.0-beta.1
generate-metadata: false
license-header: MICROSOFT_MIT_NO_VERSION
output-folder: ../src/generated
tag: package-2021-02-22-preview1
require:
- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/60be518b4fa1a9fb011a0cb69ae7ca3e1cee06b1/specification/communication/data-plane/Microsoft.CommunicationServicesTurn/readme.md
model-date-time-as-string: false
optional-response-headers: true
payload-flattening-threshold: 10
use-extension:
  "@autorest/typescript": "6.0.0-dev.20200623.2"
add-credentials: false
azure-arm: false
directive:
  from: swagger-document
  where: '$.paths["/turn/{id}/:issueCredentials"].post'
  transform: >
    $["operationId"] = "CommunicationNetworkTraversal_IssueTurnCredentials";
directive:
  rename-model:
    from: CommunicationTurnCredentialsResponse
    to: CommunicationRelayConfiguration2
```
or like this 

```yaml
package-name: azure-communication-network-traversal
override-client-name: NetworkRelayRestClient
description: Communication Network Traversal Client
package-version: 1.0.0-beta.1
generate-metadata: false
license-header: MICROSOFT_MIT_NO_VERSION
output-folder: ../src/generated
tag: package-2021-02-22-preview1
require:
- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/60be518b4fa1a9fb011a0cb69ae7ca3e1cee06b1/specification/communication/data-plane/Microsoft.CommunicationServicesTurn/readme.md
model-date-time-as-string: false
optional-response-headers: true
payload-flattening-threshold: 10
use-extension:
  "@autorest/typescript": "6.0.0-dev.20200623.2"
add-credentials: false
azure-arm: false
directive:
  from: swagger-document
  where: '$.paths["/turn/{id}/:issueCredentials"].post'
  transform: >
    $["operationId"] = "CommunicationNetworkTraversal_IssueTurnCredentials";
  rename-model:
    from: CommunicationTurnCredentialsResponse
    to: CommunicationRelayConfiguration2
```
But neither worked, so I'll add that for a separate PR.